data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -276,30 +272,9 @@
     # temporarily update this so that it's the Narwhal score that gets visualized on the map
     # The NEW final score value INCLUDES the adjacency index.
     field_names.FINAL_SCORE_N_BOOLEAN: "SN_C",
-    field_names.SCORE_N_COMMUNITIES
```

Comment:
I am so behind what we decided to do on the front end so i cannot give you a good appraisal of this!